nextflow.config

```diff
@@ -102,130 +99,130 @@ includeConfig 'conf/modules.config'
+    }
+    singularity {
+        singularity.enabled = true
+        singularity.autoMounts = true
+        docker.enabled = false
+        podman.enabled = false
+        shifter.enabled = false
+        charliecloud.enabled = false
+    }
+    podman {
+        podman.enabled = true
+        singularity.enabled = false
+        docker.enabled = false
+        shifter.enabled = false
+        charliecloud.enabled = false
+    }
+    shifter {
+        shifter.enabled = true
+        singularity.enabled = false
+        docker.enabled = false
+        podman.enabled = false
+        charliecloud.enabled = false
+    }
+    charliecloud {
+        charliecloud.enabled = true
+        singularity.enabled = false
+        docker.enabled = false
+        podman.enabled = false
+        shifter.enabled = false
+    }
+    test { includeConfig 'conf/test.config' }
+    test_full { includeConfig 'conf/test_full.config' }
+    test_tcr { includeConfig 'conf/test_tcr.config' }
 }
 
 // Export these variables to prevent local Python/R libraries from conflicting with those in the container
 env {
-  PYTHONNOUSERSITE = 1
-  R_PROFILE_USER = "/.Rprofile"
-  R_ENVIRON_USER = "/.Renviron"
+    PYTHONNOUSERSITE = 1
+    R_PROFILE_USER   = "/.Rprofile"
+    R_ENVIRON_USER   = "/.Renviron"
 }
 
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
 def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
-  enabled = true
-  file = "${params.tracedir}/execution_timeline_${trace_timestamp}.html"
+    enabled = true
+    file    = "${params.tracedir}/execution_timeline_${trace_timestamp}.html"
 }
 report {
-  enabled = true
-  file = "${params.tracedir}/execution_report_${trace_timestamp}.html"
+    enabled = true
+    file    = "${params.tracedir}/execution_report_${trace_timestamp}.html"
 }
 trace {
-  enabled = true
-  file = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
+    enabled = true
+    file    = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
 }
 dag {
-  enabled = true
-  file = "${params.tracedir}/pipeline_dag_${trace_timestamp}.svg"
+    enabled = true
+    file    = "${params.tracedir}/pipeline_dag_${trace_timestamp}.svg"
 }
 
 manifest {
-  name = 'nf-core/bcellmagic'
-  author = 'Gisela Gabernet, Simon Heumos, Alexander Peltzer'
-  homePage = 'https://github.com/nf-core/bcellmagic'
-  description = 'B and T cell repertoire analysis pipeline with the Immcantation framework.'
-  mainScript = 'main.nf'
-  nextflowVersion = '>=21.04.1'
-  version = '2.0.0'
+    name            = 'nf-core/bcellmagic'
+    author          = 'Gisela Gabernet, Simon Heumos, Alexander Peltzer'
+    homePage        = 'https://github.com/nf-core/bcellmagic'
+    description     = 'B and T cell repertoire analysis pipeline with the Immcantation framework.'
+    mainScript      = 'main.nf'
+    nextflowVersion = '!>=21.04.1'
```

Comment:
Is this right? not greater/equal to the newest release?

Reply:
21.04.1 is the latest stable - 21.06 is an edge release I think ...

Reply:
Ah yes makes sense, I somehow thought about this differently. Lower threshold then is implictly the first stable nextflow release that supported DSL2...